libs/gi/solver/src/type.d.ts

```diff
@@ -71,7 +71,7 @@ export interface CountResult {
 export interface FinalizeResult {
   resultType: 'finalize'
   builds: SolverBuild[]
-  plotData?: PlotData
```

Comment:
nit, probably dont need that since exactOptionalPropertyTypes is disabled. but no need to change it either